code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -9515,9 +9631,9 @@ public void setStat(PCStat stat, int value)
 		statValueFacet.set(id, stat, value);
 	}
 
-	public Integer getStat(PCStat stat)
```

Comment:
Which, I might add, is one of the reasons I don't turn off automated boxing warnings on my system, and why I explicitly write boxing/unboxing.  Automation would hide the warning and allow you to write methods that would basically be the equivalent of: Integer foo = Integer.valueOf(getStuff().intValue());
  ... where the valueof is implied in the foo assignment and the .intValue is implied in the return from getStuff